src/CSharp/EasyMicroservices.FileManager.AzureStorageBlobs/Interfaces/IAzureStorageBlobs.cs

```diff
@@ -0,0 +1,32 @@
+ 
+using EasyMicroservices.FileManager.AzureStorageBlobs.Models;
+using Microsoft.AspNetCore.Http;
+
+namespace EasyMicroservices.FileManager.AzureStorageBlobs.Interfaces;
+
+public interface IAzureStorageBlobs
```

Comment:
We don't need to implement new interface structure, we need to use core interface.